CLAUDE.md

```
@@ -5,14 +5,19 @@
-# or on macOS (no cat -v):
-python3 -c "print(open('path/to/file').read().splitlines()[LINE-1].__repr__())"
-```
-Then reproduce those exact characters in `old_string`. If `old_string` still fails, use a shorter match that avoids leading whitespace.
+  **Rule:** Copy the EXACT whitespace that appears after the `→` marker.
+  - Whatever appears between `→` and the code text is what's actually in the file
+  - That whitespace must be used EXACTLY in Edit tool's old_string
+  - Don't count arrows, don't interpret - just copy what's after the `→`
+
+  **Example:**
+  14→		private byte tag;
+  For Edit, use: `		private byte tag;` (copy everything after →, including the two tabs)
```

markdownlint-cli2 (0.22.0)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)